doc/multi-tenancy-vs-federation.md

```diff
@@ -0,0 +1,14 @@
+Wikibase Cloud's uses a [Multitenancy](https://en.wikipedia.org/wiki/Multitenancy) approach to many of its services.
+
+This is done to keep the cost of running each individual Wikibase low. It also reduces the costs of managing the releases of new features.
+
+We use a single instance of Blazegraph for the Wikibase instances we manage. We use the namespace feature to provide segregation of the tenant's data at the database level. We have implemented a custom service to route queries to the correct namespace. We have implemented a customer updater pipeline to allow the queryservice to be updated efficiently. This multi-tenant model is not [federation](https://www.mediawiki.org/wiki/Wikibase/Federation).
+
+We do allow federated querying between Wikibase Cloud instances. We also allow federated querying to Wikidata. We also allow federated querying to the same 3rd party endpoints that Wikidata allows.
```

Comment:
`We allow federated querying between Wikibase Cloud instances, also federated querying to Wikidata and federated querying to the same 3rd party endpoints that Wikidata allows.`

I suggest we merge these sentences into one.